lang/pt.js

```diff
@@ -7,16 +7,16 @@ export default {
 	"components.calendar.selected": "Selecionado.",
 	"components.calendar.show": "Mostrar {month}",
 	"components.dialog.close": "Fechar esta caixa de diálogo",
-	"components.form-element.defaultError": "{label} é inválido.",
+	"components.form-element.defaultError": "Valor inválido para {label}.",
 	"components.form-element.defaultFieldLabel": "Campo",
 	"components.form-element.input.email.typeMismatch": "E-mail inválido",
 	"components.form-element.input.number.rangeError": "O número deve estar entre {min} e {max}.",
 	"components.form-element.input.number.rangeOverflow": "O número deve ser menor que {max}.",
 	"components.form-element.input.number.rangeUnderflow": "O número deve ser maior que {min}.",
 	"components.form-element.input.text.tooShort": "{label} precisa ter, pelo menos, {minlength} caracteres",
 	"components.form-element.input.url.typeMismatch": "URL inválido",
-	"components.form-element.valueMissing": "{label} é obrigatório.",
-	"components.form-error-summary.errorSummary": "{count, plural, one { Foi encontrado 1 erro } other { Foram encontrados { count } erros }} nas informações que você enviou",
+	"components.form-element.valueMissing": "O campo {label} é obrigatório.",
```

Comment:
Before: `{label} is required.`
After: `The {label} field is mandatory.`

Revert